rdt-open-reader/src/test/java/io/ona/rdt/robolectric/shadow/SQLiteDatabaseShadow.java

```diff
@@ -19,5 +20,10 @@
     public static synchronized void loadLibs (Context context) {
 
     }
+
+    @Implementation
+    protected Cursor rawQueryWithFactory(SQLiteDatabase.CursorFactory cursorFactory, String sql, String[] selectionArgs, String editTable) {
```

Comment:
Why is this needed?

Reply:
`RDTSettingsSyncIntentServiceTest > testOnHandleIntentShouldCallSuperAndInitiateImageUpload` test failed due to the modification of `RDTSyncIntentService` if i won't make shadow so it failed with the message of `Password has not been set`

Reply:
Try adding https://github.com/onaio/rdt-standard/blob/07f2f1f889385f9e5bfd96da9b869c60d1e71356/rdt-open-reader/src/test/java/io/ona/rdt/robolectric/shadow/SQLiteOpenHelperShadow.java#L43 and https://github.com/onaio/rdt-standard/blob/07f2f1f889385f9e5bfd96da9b869c60d1e71356/rdt-open-reader/src/test/java/io/ona/rdt/robolectric/shadow/SQLiteOpenHelperShadow.java#L48 instead.

Reply:
I think any method dealing with that error can be handled [here](https://github.com/onaio/rdt-standard/blob/687c7969d796a669383730c9bb1d6b76e0ac9b50/rdt-open-reader/src/test/java/io/ona/rdt/robolectric/shadow/SQLiteOpenHelperShadow.java#L40). Do you know which line is throwing that error?

Reply:
yeah, now it's working fine. unused class removed